stable/insights-agent/templates/release-watcher/_job.yaml

```diff
@@ -0,0 +1,41 @@
+{{- define "releasewatcher.job.spec" -}}
+restartPolicy: Never
+serviceAccountName: {{ include "insights-agent.fullname" . }}
+volumes:
+- name: output
+  emptyDir: {}
+containers:
+- name: relase-watcher-audit
```

Comment:
`s/relase/release/`

Let's just call this `release-watcher`. The `-audit` suffixes we've been adding aren't really that helpful.